### DISCUSSION TITLE
Make [leaflet.locatecontrol] compatible with `v0.82.0`

### DISCUSSION THREAD

**Discussion by ximex** [DefinitelyTyped/DefinitelyTyped#71037] [Issues with a @types package]
https://github.com/domoritz/leaflet-locatecontrol/releases/tag/v0.82.0
With the newest version the types doesn't match the lib any more

[a comment by an automated account is not shown]

**Comment by cdauth** [comment]
I see that the module exports have been added as part of #71501.

One particular thing that I have noticed is that the latest versions of leaflet.locatecontrol don't augment the `L` object anymore. You cannot use `L.Control.Locate` or `L.control.locate` anymore. However, the typings still contain those.

**Comment by cdauth** [comment]
Another problem with the new types: The description of the `clickBehavior` option says: “Possible values are `stop` and `setView`, or the name of a behaviour to inherit from”, but the typings allow only `inViewNotFollowing: "inView"` inheritance, not the others.
